extensions/podman/packages/extension/src/utils/rosetta.ts

```
@@ -16,12 +16,74 @@
+ * Returns true when all conditions for the /etc/containers/enable-rosetta
+ * opt-in are satisfied: macOS Tahoe+ ARM64 with Rosetta enabled and Podman 5.6+.
+ * Does not require the machine to be running — safe to call before starting it.
+ */
+export async function needsRosettaEnableFile(
+  podmanConfiguration: PodmanConfiguration,
+  podmanVersion: string,
+): Promise<boolean> {
+  if (!env.isMac || arch() !== 'arm64') return false;
+
+  // The Apple-side fix only landed in macOS Tahoe (Darwin 25.x / macOS 26)
+  const darwinMajor = parseInt(release().split('.')[0], 10);
```

Biome (2.4.9)

[error] Use Number.parseInt instead of the equivalent global.

(lint/style/useNumberNamespace)